7-yoda/1-example.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+class Order
+{
+    public function updateOrder()
+    {
+        // Typical conditional structure
```

Comment:
I simply have an example class. so no need to switch to different classes since this is a very small thing